Sources/TABResourceLoader/Operations/ResourceOperation.swift

```diff
@@ -9,15 +9,15 @@
 import Foundation
 
 /// Operation used for the sole purpose of fetching a resource using a service
-public final class ResourceOperation<T: ResourceServiceType>: BaseAsynchronousOperation {
+public final class ResourceOperation<ResourceService: ResourceServiceType>: BaseAsynchronousOperation {
 
-  public typealias ResourceService = T
   public typealias DidFinishFetchingResourceCallback = (ResourceOperation<ResourceService>, Result<ResourceService.Resource.Model>) -> Void
 
   // These properties are internal for unit testing purposes
   let resource: ResourceService.Resource
   let service: ResourceService
   let didFinishFetchingResourceCallback: DidFinishFetchingResourceCallback
+  weak private(set)var cancellable: Cancellable? // Operation should not retain the cancellable item as it does not own it
```

Comment:
Interesting, did linting not pick up on there not being a space before `var`? Not sure it's even an issue or whether it's intended, just curious